### DISCUSSION TITLE
Inquiry Regarding oneDNN Warning in DIPY Codebase

### DISCUSSION THREAD

**Discussion by tanishka321** [dipy/dipy#3130] [3. Community Lounge]
While analysing the code-base I came across this part of code:(29th line)

![ss1](https://github.com/dipy/dipy/assets/123478223/1fdf9b8a-bb7a-427a-9c29-3c7781905b58)
https://github.com/dipy/dipy/blob/master/doc/examples/histology_resdnn.py

I encountered difficulty in understanding the rationale behind the decision to disable the warning associated with oneDNN as oneDNN is primarily known for its optimizations for deep learning workloads.

Thank you so much in advance for your help.

Best regards,
Tanishka Nama

**Comment by pjsjongsung** [comment]
I am not so familiar with the warnings they would throw, but wouldn't it only suppress the warnings and not the optimization it provides, if there is one? My guess is that it just threw too many warnings that hindered the view of other important output/warnings.
